src/react/components/_academics/EnrollmentLinks/Add.js

```diff
@@ -13,10 +16,11 @@ const Add = (props) => {
   } = props.instruction;
 
   const link = props.instruction.links.addEnrolledClasses;
+  const disabled = props.disabled;
 
   if (scheduleAvailable && !endOfDropAdd && enrollmentRole !== 'concurrent') {
     return (
-      <APILink {...link} name="Add" />
+      <APILink {...link} disabled={disabled} name="Add" />
```

Comment:
I'm starting to appreciate how PropTypes allows us to code things without worrying about the value type.